index.html

```diff
@@ -72,7 +72,7 @@ <h1 class="title">The One Programming Language</h1>
           </div>
         </div>
         <div class="starter-button">
-          <button type="button" id="start-learning-btn">Start Learning</button>
+          <button type="button" class="start-learning" id="start-learning">Start Learning</button>
```

Comment:
Why do we need both `class` and `id` ?

Reply:
in that class is for your styling and id for your javascript procedure i use that class just because of learning button hover glow . that class you used before in your project for that effect and it's ok to be used both id and class .